### ISSUE TITLE
v0.2.3 - broken nested list

### ISSUE THREAD

**Issue by joriverm** [ClosedXML/ClosedXML.Report#251]
Hi,

For an internal tool we have used ClosedXml to generate excel documents using a template and after some testing and trying we got our template to work.

however, recently we did an update on all our nuget packages and noticed that the excel document broke under certain circumstances.

i seem to have figured out some stuff, but i am stuck in getting it all to work correctly again.
in our template we have 3 tables , each that can have 0 or more lines. 
currently, i need to have a sum of at least 3 lines of the technologies & packages tables or the support categories tables are broken and acting very strange. however, if i have 3 lines in the tables above it, it works correctly ...

i am currently at a loss.
reverting back to 0.2.2 makes it generate the excel correctly again. did something change?

I have included the template : [ContractProposalTemplate.xlsx](https://github.com/ClosedXML/ClosedXML.Report/files/7311946/ContractProposalTemplate.xlsx)
and i have created an anonymous model class to fill in the data of the template : 
```
var templateModel = new
{
    DailyRate = 561d,
    HealthCheckPrice = 0d,
    StartupPrice = 500d,
    MonitorSetupPrice = 0d,
    CategoryDescription = "Forfait 8/18 + SLM",
    CategoryPrice = 400d,
    Technologies = new[] 
    { 
        new { Description = "Bata", MaxService = "8-18" }, 
        new { Description = "Check", MaxService = "8-18" } 
    },
    Packages = new object[]
    { 
      new { Description = "Firewall", DailyRate = 187d, DaysToInvoice = 4 }  
    },
    SupportCategories = new []
    {
        new 
        { 
            Description = "Corrective",
            Items = new [] 
            {
                new
                {
                    SupportDescription = "Normal Prio",
                    Type = "T&M",
                    DailyRateProcentage = 12.5d
                },
                new
                {
                    SupportDescription = "Medium Prio",
                    Type = "T&M",
                    DailyRateProcentage = 12.5d
                },
                new
                {
                    SupportDescription = "High Prio",
                    Type = "T&M",
                    DailyRateProcentage = 12.5d
                },
                new
                {
                    SupportDescription = "Very High Prio",
                    Type = "T&M",
                    DailyRateProcentage = 12.5d
                }
            }
        },
        new
        {
            Description = "Adaptive",
            Items = new []
            {
                new
                {
                    SupportDescription = "Urguent",
                    Type = "T&M",
                    DailyRateProcentage = 15d
                },
                new
                {
                    SupportDescription = "Standard",
                    Type = "T&M",
                    DailyRateProcentage = 15d
                },
            }
        }
    }
};
```
as is, this model creates the template correctly. however, if you comment out the package line ( new { Description = "Firewall", DailyRate = 187d, DaysToInvoice = 4 }   ) and make the packages array empty, the template will break.

any idea what is wrong?

**Comment by Pankraty** [comment]
Will be fixed in 0.2.9. Thank you for reporting and sorry it took so long to fix )

**Comment by joriverm** [comment]
no problem, this is used for an internal tool anyway ^^;
happy to see it fixed!
